test/experiments/KiD_driver/parse_commandline.jl

```diff
@@ -13,15 +13,11 @@ function parse_commandline()
         arg_type = String
         default = "Float64"
         "--moisture_choice"
-        help = "Mositure model choice: EquilibriumMoisture, NonEquilibriumMoisture"
+        help = "Mositure model choice: EquilibriumMoisture, NonEquilibriumMoisture, CloudyMoisture"
         arg_type = String
         default = "NonEquilibriumMoisture"
-        "--prognostic_vars"
```

Comment:
I'm very happy to commit to one prognostic variable choice. Do you have a preference for theta or T? We should delete the other from CI and tendency options too

Reply:
I prefer T, because in PySDM KiD and also in Shipway and Hill T is fixed.

Reply:
Agreed on T